src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_GetAllOffenderManagersAPITest.java

```diff
@@ -132,6 +135,9 @@ void canGetAllOffenderManagersByCrn() {
             assertThat(communityOffenderManager.getTeam()).isNotNull();
             assertThat(communityOffenderManager.getStaffCode()).isEqualTo("SH0007");
             assertThat(communityOffenderManager.getFromDate()).isEqualTo(LocalDate.of(2018, Month.MAY, 4));
+            assertThat(communityOffenderManager.getGrade()).isNotNull();
+            assertThat(communityOffenderManager.getGrade().getDescription()).isEqualTo("Accepted");
```

Comment:
Hmm, this looks like an odd grade, what has happened here?

Reply:
Its how the officer was created in the db https://github.com/ministryofjustice/community-api/blob/main/src/main/resources/db/data/V1_4__offenders_licences_data.sql#L250 where their STAFF_GRADE_ID was set to the reference number 1, happy to add in a script which sets the officer to an actual grade and assert on that?